packages/react-components/src/organisms/QuestionsSection.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { UserResponse } from '@asap-hub/model';
+import css from '@emotion/css';
+
+import { Card, Headline2, Headline3, Divider } from '../atoms';
+
+type ProfileSkillsProps = Pick<UserResponse, 'firstName' | 'questions'>;
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const ProfileSkills: React.FC<ProfileSkillsProps> = ({
+  firstName,
+  questions = [],
+}) => {
+  return (
+    <Card>
+      <Headline2 styleAsHeading={3}>
+        {firstName ? `${firstName}'s Open Questions` : 'Open Questions'}
+      </Headline2>
+      <div css={containerStyles}>
+        {questions.reduce((acc, question, idx) => {
```

Comment:
So I think the easiest way I've found to do this (IMO the one we should use) is
```
questions.flatMap(question => [/*Divider*/, /*Question*/]).slice(1)
```
which should be vastly shorter than this